webhook.go

```diff
@@ -174,8 +175,73 @@ func loadtsiMutateConfig(configFile string) (*tsiMutateConfig, error) {
 	return &cfg, nil
 }
 
+// isSafe validates the format of the request to check if the requested
+// operation is permitted. For CREATE operations, it checks if the request
+// contains references to proteced socket and /etc Volumes.
+// And since UPDATE, does not allowed volume modifications, it only makes sure
+// that sidecar image has not been modified
+// it returns TRUE if operation is safe or FALSE with error explanation otherwise
+func isSafe(pod *corev1.Pod, operationType string) (bool, error) {
+
+	// this output can be used for creating tests/FakeIsSafeXXX.json files
+	glog.Infof("isSafe log. Operation %v", operationType)
+	logJSON("FakeIsSafeX.json", pod)
+
+	result := errors.New("")
+	vols := pod.Spec.Volumes
+
+	switch operationType {
+	case "CREATE":
+		for _, v := range vols {
+			// glog.Infof("***** VNAME: %v VHOSTPATH: %v", v.Name, v.HostPath)
+			if v.HostPath != nil && v.HostPath.Path == "/tsi-secure/sockets" {
```

Comment:
let's check if this is prefixed by `/tsi-secure` should be a good check, since they can still mount `/tsi-secure`